service/dap/server.go

```diff
@@ -2250,16 +2373,19 @@ func (s *Server) onRestartRequest(request *dap.RestartRequest) {
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
-// onStepBackRequest sends a not-yet-implemented error response.
-// Capability 'supportsStepBack' is not set 'initialize' response.
-func (s *Server) onStepBackRequest(request *dap.StepBackRequest) {
-	s.sendNotYetImplementedErrorResponse(request.Request)
+// onStepBackRequest performs a ReverseNext command call, returning a NextResponse.
```

Comment:
It doesn't send a NextResponse.
No need to describe in English what the simple code already says in Go. How about just:

// onStepBackRequest `handles 'stepBack' request.
// This is an optional request enabled by capability ‘supportsStepBackRequest’.